.appveyor.yml

```diff
@@ -0,0 +1,26 @@
+cache:
+  - c:\tools\vcpkg\installed\
+
+install:
+  - vcpkg install boost eigen3 flann qhull
```

Comment:
If <code>vcpkg update</code> is available, I think that it able to be add vtk to installation list.
vtk port stabilized with recent commits.

Reply:
The biggest struggle here is to manage to build everything under 1 hour. You might be able to do it, but we might not be able to perform the build job under that time.

Reply:
Some dependent libraries of VTK port are optional now.
For example, Qt5 has become an option. It was took a lot of time to build.
Now, It is not necessary, because VTK port build without Qt5 by default.
Probably, It has cut down a lot of time.